src/java_tools/junitrunner/java/com/google/testing/coverage/BUILD

```diff
@@ -77,7 +77,7 @@ genrule(
     ],
     outs = ["JacocoCoverage_jarjar_deploy.jar"],
     cmd = "\n".join([
-        "JARJAR=\"$$(mktemp)\"",
+        "JARJAR=\"$$(mktemp -t bazel)\"",
```

Comment:
When I try to run this on Linux (Ubuntu 14.04) I get this:

```bash
philwo@philwo:~$ mktemp -t bazel
mktemp: too few X's in template 'bazel'
```

What about this version:
`JARJAR=\"$$(mktemp 2> /dev/null || mktemp -t bazel)\"",`